freestyle/shared/src/test/scala/freestyle/implicits.scala

```diff
@@ -41,7 +41,7 @@ class implicitsTests extends WordSpec with Matchers {
     "enable sequence" in {
       implicit val optionHandler = interps.optionHandler1
       val s = SCtors1[SCtors1.Op]
-      val program = List(s.x(1), s.x(2)).sequence[FreeS[SCtors1.Op, ?], Int]
+      val program = List(s.x(1), s.x(2)).sequence[FreeS.Par[SCtors1.Op, ?], Int]
```

Comment:
Another point in which, by being now parallel by default, we have to make a special change.